---
layout: page
title: About me
---

## Who ?

I am Olly Burren currently a PhD student within the prestigious [Diabetes and Inflammation (DIL) Laboratory](https://www-gene.cimr.cam.ac.uk/) at the [University of Cambridge](http://www.cam.ac.uk/). I am supervised by <a href="https://en.wikipedia.org/wiki/John_A._Todd_(biologist)">Professor John Todd</a> with statisticical and computatioonal co-supervision from <a href="http://chr1swallace.github.io/">Chris Wallace</a>, Wellcome Trust senior research fellow, Department of Medicine  and Leonardo Bortollo, Reader in Statistics for Biomedicine, Department of Medical Genetics. Concurrently I am Head of Genome Informatics at the DIL, and am lucky to manage a team of three very talented Bioinformaticians, who are responsible for [ImmunoBase](http://www.immunobase.org) and [T1DBase](http://www.t1dbase.org), two widely used and cited human autoimmune genetics databases. 

## Where ?

The DIL, is an incredibly stimulating  multi-discipline organisation and as such I am extremely lucky to work with experts in encompassing Immunology, Experimental Medicine and Biostatistics. It gets better, however, as we are situated within the CIMR on the Cambridge Biomedicine Campus, which itself is close to the [MRC BSU](http://www.mrc-bsu.cam.ac.uk/), [WTSI](http://www.sanger.ac.uk/), [EBI](http://www.ebi.ac.uk),[Babraham Institute](http://www.babraham.ac.uk/) with whom I currently collaborate.

## Why ?

Our lab is particularly interested in understanding the cellular mechanisms through which autoimmunity is mediated, focussing on type 1 diabetes. This disease primarily singles out young people, and caused selective destruction of pancreatic islets cells, resulting in a lifelong dependency on exogenous insulin as well as some nasty co-morbidities. Worringly it's prevelence is increasing. It's a complex, polygenic disease  which means that there is an underlying genetic component involving lots of genes that is modulated by environmental triggers. Our lab seeks to understand both of these components, to suggest therapeutic avenues to treat existing type 1 diabeteics and primary prevention strategies to reduce prevelence. As it turns out, the genetics between autoimmune diseases is shared, and we can use this to our advantage to gain further insights other disease mechanisms and perhaps repurpose existing treatments. Understanding the Immune system and it's regulation is also a trick that we might someday use to treat cancer. On a personal level I get to work on challenging and interesting problems in an area with direct societal benefit 

## What ?

A technique called a genome wide association study (GWAS) pioneered at DIL in collboration with others, can give us a lot of information on the genetic differences that underlie a disease. However translating these into biological mechanisms has proved tricky as our knowledge of how DNA works within human cells is patchy. Fortunately the field of genomics which studies such things has some powerful tools and datasets that we can use. For the budding data scientist in me combining and aligning these datasets using the genetics presents some extremely interesting challenges.   

## How ?

A technique called a genome wide association study (GWAS) pioneered at DIL in collboration with others, can give us a lot of information on the genetic differences that underly a disease. However translating these into biological mechanisms has proved tricky. Work I have carried out previously attempted to tag genes with disease based on proximity, however after reading this paper, I realised that things were probably much more complicated. Up the road at the Babraham Institute Peter Fraser's group has been pioneering a technique called promoter capture Hi-C that gives us information about which bits of DNA bend and touch each other, even though they might be distant on a linear DNA molecule. With many others we have been working on methods to leverage these interactions in 17 primary human cell types with other genomic datasets to get a better understanding of Haematopoiesis in humans. I have been privileged to have the task to intergrate GWAS summary statistics to answer the following questions:-

  * Are GWAS signals for traits enriched within interactions, and if so is this tissue specific ?
  * Can we use this information to prioritise genes and GWAS signals for followup study ?
